src/game_api/level_api.hpp

```diff
@@ -538,14 +560,35 @@ struct LevelGenSystem
             LevelGenRooms* rooms_backlayer;
```

Comment:
Can you change the `rooms_frontlayer` to `rooms_front_layer` to align with the renames below?